dbschema/product.esdl

```diff
@@ -66,6 +66,11 @@ module default {
 }
   
 module Product {
+  type CompletionDescription {
+    query: str;
```

Comment:
`query` is the input name.
I'd call this `value` or `description`, as it is currently in the DB.
https://github.com/SeedCompany/cord-api-v3/blob/63a27a21040c8ff7c9b50efd7e6e865e35cb7a61/src/components/product/product.repository.ts#L531-L531
It's also required.

Reply:
I'm confused as to how these 2 fields are required.  On the UI you can leave them blank.

Reply:
What UI? The DB nodes always have these [two values](https://github.com/SeedCompany/cord-api-v3/blob/63a27a21040c8ff7c9b50efd7e6e865e35cb7a61/src/components/product/product.repository.ts#L524-L532).

Reply:
When you fill out a goal on the UI you can save without selecting a Methodology (or Completion Description).  And if you do choose one then you can save without having a Completion Description.  Maybe this is not the fields backing that UI on a Product?

Reply:
Yeah this is just to help suggest existing values for this field
https://github.com/SeedCompany/cord-api-v3/blob/63a27a21040c8ff7c9b50efd7e6e865e35cb7a61/dbschema/product.esdl#L15-L15

Reply:
I actually don't even know if we need this. Maybe we could just add an FTS index to that Product field.